From f4e5e74548d92258855434b550c58fe44993c148 Mon Sep 17 00:00:00 2001
From: Sumit Garg <sumit.garg@linaro.org>
Date: Tue, 20 Jul 2021 14:20:10 +0530
Subject: [PATCH] Makefile: Enable plugins installation in rootfs

Upstream-Status: Submitted [https://github.com/linaro-swg/optee_examples/pull/87]

Signed-off-by: Sumit Garg <sumit.garg@linaro.org>

---
 Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makefile b/Makefile
index a275842..9359d95 100644
--- a/Makefile
+++ b/Makefile
@@ -25,14 +25,19 @@ prepare-for-rootfs: examples
 	@mkdir -p $(OUTPUT_DIR)
 	@mkdir -p $(OUTPUT_DIR)/ta
 	@mkdir -p $(OUTPUT_DIR)/ca
+	@mkdir -p $(OUTPUT_DIR)/plugins
 	@for example in $(EXAMPLE_LIST); do \
 		if [ -e $$example/host/optee_example_$$example ]; then \
 			cp -p $$example/host/optee_example_$$example $(OUTPUT_DIR)/ca/; \
 		fi; \
 		cp -pr $$example/ta/*.ta $(OUTPUT_DIR)/ta/; \
+		if [ $$example = plugins ]; then \
+			cp -p plugins/syslog/*.plugin $(OUTPUT_DIR)/plugins/; \
+		fi; \
 	done
 
 prepare-for-rootfs-clean:
 	@rm -rf $(OUTPUT_DIR)/ta
 	@rm -rf $(OUTPUT_DIR)/ca
+	@rm -rf $(OUTPUT_DIR)/plugins
 	@rmdir --ignore-fail-on-non-empty $(OUTPUT_DIR) || test ! -e $(OUTPUT_DIR)
