diff --git a/include/configs/vexpress_aemv8a.h b/include/configs/vexpress_aemv8a.h
index 2354f4e958..3e01f477dc 100644
--- a/include/configs/vexpress_aemv8a.h
+++ b/include/configs/vexpress_aemv8a.h
@@ -151,6 +151,32 @@
 				"fdt_addr=0x83000000\0" \
 				"fdt_high=0xffffffffffffffff\0" \
 				"initrd_high=0xffffffffffffffff\0" \
+                                "bootenvfile=uEnv.txt\0" \
+                                "bootcmd=run envboot\0" \
+                                "envboot=if run loadbootenv; then echo Loading env from ${bootenvfile}; run importbootenv; else run default_bootcmd; fi; if test -n $uenvcmd; then echo Running uenvcmd ...; run uenvcmd;fi;\0" \
+                                "importbootenv=echo Importing environment from memory, size ${filesize}; env import -t ${loadaddr} ${filesize}\0" \
+                                "loadaddr=0x82000000\0" \
+                                "filesize=0x4000\0" \
+                                "loadbootenv=mw.l ${loadaddr} 0 0x1000; afs load ${bootenvfile} ${loadaddr}\0" \
+				"default_bootcmd=echo running default boot command; afs load ${kernel_name} ${kernel_addr} ; " \
+                                		"if test $? -eq 1; then "\
+		                                "  echo Loading ${kernel_alt_name} instead of "\
+                		                "${kernel_name}; "\
+		                                "  afs load ${kernel_alt_name} ${kernel_addr};"\
+                                		"fi ; "\
+		                                "afs load  ${fdtfile} ${fdt_addr} ; " \
+                		                "if test $? -eq 1; then "\
+		                                "  echo Loading ${fdt_alt_name} instead of "\
+		                                "${fdtfile}; "\
+		                                "  afs load ${fdt_alt_name} ${fdt_addr}; "\
+		                                "fi ; "\
+                		                "fdt addr ${fdt_addr}; fdt resize; " \
+		                                "if afs load  ${initrd_name} ${initrd_addr} ; "\
+		                                "then "\
+		                                "  setenv initrd_param ${initrd_addr}; "\
+		                                "  else setenv initrd_param -; "\
+		                                "fi ; " \
+		                                "booti ${kernel_addr} ${initrd_param} ${fdt_addr}\0"
 
 /* Copy the kernel and FDT to DRAM memory and boot */
 #define CONFIG_BOOTCOMMAND	"afs load ${kernel_name} ${kernel_addr} ; " \
